BoardGameProject/src/main/java/com/team4/model/classes/King.java

```diff
@@ -32,4 +32,17 @@ public List<Space> getPossibleMoves() {
         // We gotta add some code here
         return possibleMoves;
     }
-}
\ No newline at end of file
+<<<<<<< HEAD
+}
+=======

```

Comment:
this looks like a merge conflict, could you handle this?
Same for the Pawn class